hawkular-command-gateway-api/src/main/resources/schema/A0uthMessage.schema.json

```diff
@@ -10,15 +10,39 @@
         "token":    { "type": "string" },
         "persona":  { "type": "string" }
       }
+    },
```

Comment:
we should name this file "A0AuthMessage.schema.json" - since we now have others "A1..." and "A2"... seems like we should just prefix "AuthMessage" with "A0" to match that pattern. A0thMessage just looks weird now :)